webapp/model/Dispute.php

```diff
@@ -30,6 +31,10 @@ public function getState($account = false) {
         return DisputeStateCalculator::getState($this, $account);
     }
 
+    public function getMediationState($account = false) {
+        return $this->mediationState;
```

Comment:
Unit tests:
* getMediationState of different (known) disputes.
* assert the $medState->isInMediation() is false, etc.